src/arrays/ArrayUtil.java

```diff
@@ -11,85 +11,167 @@
+        System.out.println("addFirst");
 
-        return null;
+        int[] arr = new int[array.length];
+        arr[0] = valueToAdd;
+        System.arraycopy(arr, 0, array, 0, 1);
+        return array;
     }
 
+
     /**
      * Implement a method that will check if the given value contains in the array
-     *
-     * @return true if the array contains the given value, otherwise return false.
      */
-    public static boolean contains(int[] array, int value) {
-
+    public static boolean contains(int[] array1, int value) {
+        System.out.println("contains");
+
+        for (int i : array1) {
+            if (value == array1[i]) {
+                return true;
+            }
+        }
         return false;
     }
 
     /**
-     * @param array An array to search in.
+     * @param array2 An array to search in.
      * @param value The value to search for.
      * @return the first index of value in the array, if array does not contain given element return -1.
      */
-    public static int indexOf(int[] array, int value) {
-
-        return 0;
+    public static int indexOf(int[] array2, int value) {
+        System.out.println("indexOf");
+
+        for (int i : array2) {
+            if (value == array2[i]) {
+                return value;
+            }
+        }
+        return -1;
     }
 
     /**
      * Remove the element by the given index.
      * Print new array
      */
-    public static void remove(int[] array, int index) {
-
+    public static void remove(int[] array3, int index) {
+        System.out.println("remove");
+
+        int[] arr = new int[array3.length - 1];
+        for (int i : array3) {
+            System.out.print(i + ", ");
+        }
+        System.out.println();
+        if (index > array3.length - 1) {
+            System.out.print("index chka");
+        } else {
+            for (int i = 0; i < array3.length - 1; i++) {
+                if (i < index) {
+                    arr[i] = array3[i];
+                } else {
+                    arr[i] = array3[i + 1];
+                }
+            }
+            for (int i : arr) {
+                System.out.print(i + ", ");
+            }
+        }
+        System.out.println();
+        System.out.println("_______________________");
     }
 
 
     /**
      * Calculate and return sum of array's elements.
      */
-    public static long sum(int[] array) {
-
-        return 0;
+    public static long sum(int[] array5) {
+        System.out.println("sum");
+
+        int sum = 0;
+        for (int i : array5) {
+            sum += array5[i];
+        }
+        return sum;
     }
 
 
     /**
      * Get the max value from the array.
      */
-    public static int getMax(int[] array) {
-
-        return 0;
+    public static int getMax(int[] array6) {
+        System.out.println("getMax");
+
+        int max = array6[0];
+        for (int i : array6) {
+            if (max < array6[i]) {
+                max = array6[i];
```

Comment:
If you want to use INDEXes use FOR loop, not FOREACH.
Change your array elements values and you will see what is wrong.